internal/pkg/filesystem/aferofs/utils.go

```diff
@@ -39,9 +41,10 @@ func CopyFs2Fs(srcFs filesystem.Fs, srcPath string, dstFs filesystem.Fs, dstPath
 
 	// nolint: forbidigo
 	return aferocopy.Copy(srcPath, dstPath, aferocopy.Options{
-		SrcFs:  aferoSrc,
-		DestFs: aferoDst,
-		Sync:   true,
+		SrcFs:          aferoSrc,
+		DestFs:         aferoDst,
+		Sync:           false,
+		CopyBufferSize: CopyBufferSize,
```

Comment:
Skumal som, ze API je teraz nejake pomale.

Prva vec, co som nasiel je, ze `dev/beta` branch nenacitavame automaticky pri starte servera, ale az pri prvom requeste (to je upravene v dalsom PR), ... ale ze nacitanie repository trva `10-20s` (aj automaticky update cez cron, ale to nevadi).


Zistil som, ze ked tu vypnem `sync` a dam vacsi buffer, tak su to uz iba `2sec`.
Sync garantuje, ze su udaje zapisane na disku, nie iba v OS cache, co myslim, nikde v nasom projekte nepotrebujeme.

Reply:
Aha, super 👍